src/compute/src/render/mod.rs

```diff
@@ -144,6 +147,31 @@ pub fn build_compute_dataflow<A: Allocate>(
     compute_state: &mut ComputeState,
     dataflow: DataflowDescription<Plan, CollectionMetadata>,
 ) {
+    fn intercept_source_instantiation_frontiers<G>(
```

Comment:
You could move this function outside of `build_compute_dataflow` to avoid nesting.